lib/v2/datasource/remote/api/profile_repository.dart

```diff
@@ -20,4 +20,36 @@ class ProfileRepository extends NetworkRepository {
             }))
         .catchError((error) => mapError(error));
   }
+
+  // Future<dynamic> updateProfile({
```

Comment:
is this commended code needed?

Reply:
Again,  I'm actually moving this forward on the Edit Name Screen branch, I just foget remove it here.